website/content/docs/enterprise/pkcs11-provider/aws-xks.mdx

```diff
@@ -0,0 +1,201 @@
+   tls_cert_pem = "tls/server_cert.pem"
+   tls_key_pem = "tls/server_key.pem"
+   mtls_client_ca_pem = "tls/client_ca.pem"
+   mtls_client_dns_name = "us-east-1.alpha.cks.kms.aws.internal.amazonaws.com"
+
+   [[external_key_stores]]
+   uri_path_prefix = "/xyz"
+   sigv4_access_id = "AKIA4GBY3I6JCE5M2HPM"
+   sigv4_secret_key = "1234567890123456789012345678901234567890123="
+   xks_key_id_set = ["abc123"]
+
+   [pkcs11]
+   session_pool_max_size = 30
+   session_pool_timeout_milli = 0
+   session_eager_close = false
+   user_pin = ""
+   PKCS11_HSM_MODULE = "/usr/local/lib/libvault-pkcs11.so"
+   context_read_timeout_milli = 100
+
+   [limits]
+   max_plaintext_in_base64 = 8192
+   max_aad_in_base64 = 16384
+
+   [hsm_capabilities]
+   can_generate_iv = false
+   is_zero_iv_required = false
+   ```
+
+1. Create a file, `/etc/vault-pkcs11.hcl` with the following contents:
+
+    ```hcl
+    slot {
+      server = "VAULT_ADDRESS:5696"
+      tls_cert_path = "/etc/kmip/cert.pem"
+      ca_path = "/etc/kmip/ca.pem"
+      scope = "my-service"
+    }
+    ```
+
+    This file is used by `libvault-pkcs11.so` to know how to find and communicate with the KMIP server.
+    See [the Vault docs](https://developer.hashicorp.com/vault/docs/enterprise/pkcs11-provider) for all available parameters and their usage.
+
+1. If you want to view the Vault logs (helpful when trying to find error messages), you can specify the `VAULT_LOG_FILE` (default is stdout) and `VAULT_LOG_LEVEL` (default is `INFO`). We'd recommend setting `VAULT_LOG_FILE` to something like `/tmp/vault.log` or `/var/log/vault.log`. Other useful log levels are `WARN` (quieter) and `TRACE` (very verbose, could possibly contain sensitive information, like raw network packets).
+
+## Enable XKS in the AWS UI
+
+1. Go into KMS settings in the AWS UI.
+1. Go to "External key stores" in the navigation menu.
+1. Click on "Create external key store"
+1. Add the relevant settings:
+   a. Key store name: the name you want your key store to be referred to in AWS
+   a. Proxy URI endpoint: the base URI of the XKS proxy server (e.g., `https://xks.example.com:1234`)
+   a. Proxy URI path prefix: added to the end of the base URI of the XKS proxy server when making requests
+   a. Proxy credential, Access key ID: the access key you configured in the XKS proxy configuration file
+   a. Proxy credential, Secret access key: the secret key you configured in the XKS proxy configuration file
+1. Create the external key store
+1. Connect the external key store (will take a few minutes)
+1. Select the newly created external key store
+1. Select "Create a KMS key in this key store"
+   a. Enter an external key ID corresponding to a PKCS#11 label that is allowed in the settings TOML. In the example above, this is "abc123".
+   a. Enter an alias for this key.
+   a. Set up the key administrators and key usage permissions.
+1. You should not be able to use this Vault-backed key with the KMS API, e.g., `aws kms encrypt`.
```

Comment:
Should this say `now` instead of `not`? Or maybe there's some other way to use the key?

```suggestion
1. You should now be able to use this Vault-backed key with the KMS API, e.g., `aws kms encrypt`.
```